adagrams/game.py

```diff
@@ -1,11 +1,154 @@
+import random 
+import pdb
+
 def draw_letters():
-    pass
+
+    LETTER_POOL = {
```

Comment:
Things declared with all upper case are considered constants, which means they should not change. However, in the code below, this data structure is modified. That means `LETTER_POOL` should be `letter_pool` instead.